### ISSUE TITLE
[BUG] Not rendering properly when container is specified with float height

### ISSUE THREAD

**Issue by Elabar** [petyosi/react-virtuoso#444]
**Describe the bug**
The list fail to render correctly (not rendering anything) when the container height is set to floating number and `initialTopMostItemIndex` is specified as well.

**Reproduction**
https://stackblitz.com/edit/react-jbwg9m?file=src%2FApp.js

**To Reproduce**
I have put in 4 examples in the reproduction link.

**Expected behavior**
The list should render correctly or document this problem in troubleshooting section.

**Screenshots**
Please check the reproduction.

**Comment by petyosi** [comment]
Question - how come you end up with a float height? I've been debugging a similar problem with item heights lately.

**Comment by Elabar** [comment]
# TLDR;
Wanted to throw this list into a modal and set the modal height to 70vh

# Story
Was implementing this virtualization in a [modal](https://headlessui.dev/react/dialog) and I just set the container to 70vh.

First thing I realize is, this library is not so happy with style that have scale transformation in it, so I only render the list after the fade in + zoom in animation is finish with [`afterEnter`](https://headlessui.dev/react/transition#transition)

After the change, the list is still not rendering properly and I try to set the container height to 100 (integer) and the list does render properly. After lot of attempts, I found the pattern to reproduce it as in the [reproduction](https://stackblitz.com/edit/react-jbwg9m?file=src%2FApp.js) and here we are
